### PR TITLE
Implement YouTube quality options skeleton code

### PR DESCRIPTION
**Pull request by jdragonbae**
This patch implements skeleton code about quality options for YouTube. It will
control the quality using JavaScript.

Refs: #776

### CODE REVIEW

File: src/org/triple/banana/media_remote/MediaRemoteViewImpl.java [hunk 1]
```diff
@@ -224,6 +233,7 @@ public void dismiss() {
         if (mDialog == null || !mDialog.isShowing()) return;
 
         resetContentViewInteractionListeners();
+        YouTubeUtil.dismissQualityOptionsDialog();
```
Comment (romandev) [inline src/org/triple/banana/media_remote/MediaRemoteViewImpl.java:236]: Why do we need this? As I know, if the parent is gone, does the dialog is also dismissed automatically?

Comment (jdragonbae) [reply src/org/triple/banana/media_remote/MediaRemoteViewImpl.java:236]: When enter pip mode, the dialog remain on the media. WDYT?

Comment (romandev) [reply src/org/triple/banana/media_remote/MediaRemoteViewImpl.java:236]: Ah okay, I missed it.

**Review by romandev — APPROVED** [review]
lgtm

**Pull request merged**